docs/vSphere-prerequisites.md

```diff
@@ -0,0 +1,28 @@
+# Pre-configuration steps in case of vSphere cluster:
+## Windows VM assumptions:
+- An updated version of Windows Server 1909 VM image and corresponding container images are used
+- The Windows image is configured with SSH so that WMCO can access it:
+  - SSH on the Windows VM is configured using the [powershell script](powershell.yaml)
+  - A new file called *C:\Users\Administrator\.ssh\authorized_keys* containing the public key to be used is created
+- VMWare tools is installed, configured and running on the Windows VM:
+  - The [tools.conf](https://docs.vmware.com/en/VMware-Tools/11.1.0/com.vmware.vsphere.vmwaretools.doc/GUID-EA16729B-43C9-4DF9-B780-9B358E71B4AB.html) 
+  file at *C:\ProgramData\VMware\VMware Tools\tools.conf* has the following entry to ensure that the cloned 
+  vNIC generated on the Windows VM by hybrid-overlay won't be ignored and the VM has an IP Address in the vCenter
+    - `exclude-nics=`
+  - The `VMTools` Windows service is running in the Windows VM.
+- The Windows image has been sysprep'ed and made ready for clone using the following command:
+
+`C:\Windows\System32\Sysprep\sysprep.exe /generalize /oobe /shutdown /unattend:<path_to_unattend.xml>`
+
+Sample [unattend.xml](unattend.xml)
```

Comment:
What is this used for? It looks like its used to create a VM. Does a user have to use this one? If not saying something like
```
Optional sample [unattend.xml](unattend.xml) which can be used to create a Windows VM
```
would be helpful